cfgov/jinja2/v1/landing-page/index.html

```diff
@@ -86,6 +86,7 @@ <h2 class="o-half-width-link-blob-group__header">
 {%- endblock %}
 
 {% block content_sidebar scoped -%}
+    {{ related_posts.render() }}
     {% for block in page.sidebar %}
```

Comment:
@kurt If you want to stop receiving notifications, you can unsubscribe to this thread using the button on the right side of the page. Screenshot: http://grab.by/MrN4